src/geometry/collider.rs

```diff
@@ -492,12 +503,22 @@ impl ColliderBuilder {
         self
     }
 
-    /// Sets the density of the collider this builder will build.
+    /// Sets the uniform density of the collider this builder will build.
+    /// This will be overridden by a call to [`Self::mass_properties`] so it only makes sense to call
+    /// either [`Self::density`] or [`Self::mass_properties`].
```

Comment:
```suggestion
    /// Sets the uniform density of the collider this builder will build.
    ///
    /// This will be overridden by a call to [`Self::mass_properties`] so it only makes sense to call
    /// either [`Self::density`] or [`Self::mass_properties`].
```